cli/011-cli-multi-config.md

```diff
@@ -0,0 +1,74 @@
+- Start Date: 2021-08-22
+- RFC PR:
+- Issue:
+
+### Configuring multiple environments
+
+##### Problem
+
+Users may have multiple Temporal service environments (development, production, ..) with their own respective mTLS configs, namespaces, address etc. Current implementation of tctl config feature only supports configuring a single environment and doesn't help at quickly switching between multiple.
+
+##### Proposal
```

Comment:
During onboarding it's pretty common to have a command to copy/paste or script to run that sets up a user's development environment. Could the proposal include an example of what a script to add or update an environment would look like, if that is a supported use case from the CLI?

In particular I'm wondering if `tctl config use-environment <name>` creates a new environment if none exists -- this could lead to some config file pollution when typos happen (and is why adding and updating are separate commands in `tctx`).

Reply:
I tend to agree, there should be a separate command for creating a new environment.

Reply:
sg!